spec/services/ial_context_spec.rb

```diff
@@ -55,6 +55,56 @@
     end
   end
 
+  describe '#default_to_ial2?' do
```

Comment:
@achapm do you want to add a quick test for `ial2_or_greater?` as well (when no `ial` is passed) since that was actually the key change?

Reply:
Yes, thanks. See below